### PR TITLE
Refactor paging: show more tricks on the home page without reloading the entire page

### PR DESCRIPTION
**Pull request by nicordev**
Use of AJAX to show more tricks on the home page without reloading the entire page

### CODE REVIEW

File: src/Controller/HomeController.php [hunk 1]
```diff
@@ -35,4 +37,25 @@ public function show(ObjectManager $manager, Paginator $paginator, int $page = 1
...
+     * Get a page of tricks from an ajax call
+     *
+     * @Route("/get-page/{page}", name="home_ajax_get_page", requirements={"page": "\d+"})
+     *
+     * @param ObjectManager $manager
+     * @param Paginator $paginator
+     * @param int $page
+     * @return false|string
+     */
+    public function getPage(ObjectManager $manager, Paginator $paginator, ?int $page = null)
```
Comment (olix21) [inline src/Controller/HomeController.php]: use the `EntityManagerInterface` instead of the `ObjectManager` (interface vs implementation)

Comment (olix21) [inline src/Controller/HomeController.php]: `int $page = 1` to avoid `$page ?? 1` @line 55?

Comment (nicordev) [reply src/Controller/HomeController.php]: Done.

Comment (nicordev) [reply src/Controller/HomeController.php]: Done. Awesome, I didn't know this could work!

**Pull request merged**